docs/guides/anomaly-detection-configuration/ignore_small_changes.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "ignore_small_changes"
+sidebarTitle: "ignore_small_changes"
```

Comment:
This should also be added to the param list of all the tests (like [here](https://docs.elementary-data.com/guides/anomaly-detection-tests/volume-anomalies#test-configuration))